cnf-tests/mirror/mirror.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+)
+
+type image struct {
+	Name     string `json:"name"`
+	Registry string `json:"registry"`
+	Version  string `json:"version"`
+}
+
+func main() {
+	imagesFile := flag.String("images", "/usr/local/etc/cnf/images.json", "the json file containing the images")
+	targetRegistry := flag.String("registry", "", "the target registry we want to mirror to")
+
+	flag.Parse()
+
+	if *imagesFile == "" || *targetRegistry == "" {
+		flag.Usage()
+		log.Fatal("Missing mandatory fields")
+	}
+
+	bytes, err := ioutil.ReadFile(*imagesFile)
+	if err != nil {
+		log.Fatalf("Failed to read %s - %v", *imagesFile, err)
+	}
+
+	var images []image
+	err = json.Unmarshal(bytes, &images)
+	if err != nil {
+		log.Fatalf("Failed to read %s - %v", *imagesFile, err)
+	}
+
+	for _, img := range images {
+		fmt.Printf("%s%s:%s %s%s:%s\n", img.Registry, img.Name, img.Version, *targetRegistry, img.Name, img.Version)
+	}
+}
```

Comment:
maybe add a TODO here to add the ability to push from here and not just print this?

another solution that I think can be also fine is to have an entry in the `Makefile` that will run this container and `| oc mirror` the images?

Reply:
Yes, on the long term we may even think about using the oc command from inside the image.
Adding the makefile entry would work for local development, not when other users use the image.

I'll stick with the TODO for the time being.